webapp/api/data/session-data-defaults.js

```diff
@@ -51,3 +51,10 @@ export default {
   'salvage-services': '',
   submittedFiles: []
 };
+
+const getDefaults = () => {
+  // use JSON serialisation to create a deep clone of the defaults object
```

Comment:
I'd be tempted to define `const defaultData = ...` inside getDefaults. That way we know we're always created a new object and nested objects.

Reply:
👍 changes made in https://github.com/mcagov/droits/pull/223/commits/d54c983294c3dcbcfc81e275b728692c5d449b3a